scripts/get

```diff
@@ -48,25 +58,37 @@ if [ -n "$PKG_URL" -a -n "$PKG_SOURCE_NAME" ]; then
     sleep 1
   done
 
-  if ! [ -f $SOURCES/$1/$PKG_SOURCE_NAME -a "$(cat $STAMP 2>/dev/null)" == "$PKG_URL" ]; then
-    rm -f $SOURCES/$1/$PKG_SOURCE_NAME $STAMP
+  if ! _get_file_already_downloaded $1; then
```

Comment:
I've pushed one more update which handles `_get_file_already_downloaded()` the same way in the two places it is called - if it returns true, the script exits immediately. This has also eliminated one level of `if` indentation in subsequent lines.